configs/config.d/auth.conf

```diff
@@ -0,0 +1,8 @@
+<source>
+  type tail
+  format /^(?<message>(?<time>[^ ]*\s*[^ ]* [^ ]*) .*)$/
```

Comment:
What is the regular format like? The regex to match the `time` section with `[^ ]*` feels a bit loose... Can we do something like `\d` or `\w`?

Reply:
Good catch, I've updated it to be more strict. PTAL.